cmake/extensions.cmake

```diff
@@ -944,6 +947,11 @@ function(import_kconfig prefix kconfig_fragment)
     endif()
 
     set("${CONF_VARIABLE_NAME}" "${CONF_VARIABLE_VALUE}" PARENT_SCOPE)
+    list(APPEND keys "${CONF_VARIABLE_NAME}")
```

Comment:
You are going to need to set keys as an output variable in the PARENT_SCOPE, are you not?

Reply:
Ah, now I understand what the below foreach is doing ...